sql/localsql/0003_migration_interfaces.go

```diff
@@ -0,0 +1,12 @@
+package localsql
+
+import (
+	"context"
+)
+
+//go:generate mockgen -typed -package=localsql -destination=./0003_mocks.go -source=./0003_migration_interfaces.go
+
+type PoetClient interface {
```

Comment:
I think there's no need to export it.
```suggestion
type poetClient interface {
```

Reply:
I tried to have it private, but I need it in `node.go` (around line 1620) when I instantiate the poet clients and pass them to the migration that needs it. Since the `node` package already imports `localsql` this made sense to me but wdyt?

Reply:
I don't understand why the interface needs to be exported. The poet HTTP client from the `activation` pkg satisfies it regardless (of exported/private) so it should work. Why not?

Reply:
I tried it, but the call in `node.go` takes a slice of interfaces and Go doesn't support covariance on slices. I would need to make `New0003Migration` take `[]*activation.HTTPClient` if we don't export the interface, but that would complicate testing and requires some refactoring (to avoid cyclic dependencies).